### PR TITLE
Subscription Flows (SU5, SU6, SU8)

### PR DESCRIPTION
**Pull request by tahmidefaz**
Jiras:
* RHCLOUD-34682
* RHCLOUD-34683
* RHCLOUD-34685

### CODE REVIEW

**Review by justinorringer — APPROVED** [review]

File: data/insights/subscriptions/domain.yml [hunk 1]
```diff
@@ -74,3 +91,18 @@ responses:
...
+    - text: "Which product's usage are you looking for?"
+      buttons:
+        - title: RHEL
+          payload: rhel
+        - title: OpenShift
+          payload: openshift
+  utter_subs_product_usage_rhel_page:
+    - text: "Check out the [RHEL's usage page here](https://console.redhat.com/subscriptions/usage/rhel)."
+  utter_subs_product_usage_openshift_page:
+    - text: "Check out the [OpenShift's usage page here](https://console.redhat.com/subscriptions/usage/openshift)."
```
Comment (catastrophe-brandon) [inline data/insights/subscriptions/domain.yml:104]: Also, the wording is a little weird here. Possessive nouns are used but a simple noun would do. Maybe:

[RHEL usage page here]
[OpenShift usage page here]

File: data/insights/subscriptions/domain.yml [hunk 2]
```diff
@@ -74,3 +91,18 @@ responses:
...
         * {expiring} subscriptions that are expiring soon
         * {expired} that have already expired
+  utter_ask_form_subs_product_usage_subs_product_type:
+    - text: "Which product's usage are you looking for?"
+      buttons:
+        - title: RHEL
+          payload: rhel
+        - title: OpenShift
+          payload: openshift
+  utter_subs_product_usage_rhel_page:
```
[the remark this answers is not in the document]
Comment (tahmidefaz) [reply data/insights/subscriptions/domain.yml:101]: Ah, I thought about it, but I realized that am not that familar with subscriptions. So I am not sure what else to write here. :smile:

File: data/insights/subscriptions/domain.yml [hunk 3]
```diff
@@ -74,3 +91,18 @@ responses:
...
+    - text: "Check out the [RHEL usage page here](https://console.redhat.com/subscriptions/usage/rhel). That page will show your RHEL usage for both annual and on-demand subscriptions."
+  utter_subs_product_usage_openshift_page:
+    - text: "Check out the [OpenShift usage page here](https://console.redhat.com/subscriptions/usage/openshift). That page will show your OpenShift usage for both annual and on-Demand subscriptions."
+  utter_subs_simple_content_access_docs:
+    - text: "Check out [Simple Content Access documentation here](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_simple_content_access/index)."
+  utter_subs_activation_keys_docs:
+    - text: "Check out [Activation key documentation here](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys)."
```
Comment (justinorringer) [inline data/insights/subscriptions/domain.yml:108]:
```suggestion
    - text: "You can monitor your RHEL subscription usage on the [Subscription Usage Dashboard](https://console.redhat.com/subscriptions/usage/rhel). Usage information is available for both Annual and On-demand subscriptions."
  utter_subs_product_usage_openshift_page:
    - text: "You can monitor your Openshift subscription usage on the [Subscription Usage Dashboard](https://console.redhat.com/subscriptions/usage/openshift). Usage information is available for both Annual and On-demand subscriptions."
  utter_subs_simple_content_access_docs:
    - text: "Simple Content Access simplifies workflows for managing your subscriptions. For more information on getting started, visit the [Simple Content Access documentation here](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_simple_content_access/index)."
  utter_subs_activation_keys_docs:
    - text: "To auto-configure your systems, you'll need an activation key. For managing your tokens, visit the [Activation key documentation here](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_activation_keys_on_the_hybrid_cloud_console/assembly-creating-managing-activation-keys)."
```

Comment (justinorringer) [reply data/insights/subscriptions/domain.yml:108]: Meant to do that yesterday haha, but here ya go!

**Pull request merged**